### PR TITLE
Update ru_RU.lang

### PR DESCRIPTION
**Pull request by Johni0702**
Based on https://github.com/ReplayMod/Translations/pull/13#issuecomment-1186623261
This supersedes #10, #13 and #15 and will be merged if there if no objections have been brought up within a week or at least two people are happy with it.
If you would like to make additional changes to keys not affected by this PR, please open a separate PR.

### CODE REVIEW

File: ru_RU.lang [hunk 1]
```diff
@@ -66,7 +66,7 @@ replaymod.gui.videothumbnailtoolarge=Размер изображения пре
 replaymod.gui.videothumbnailformat=Изображение должно быть в формате JPEG или PNG
 
 replaymod.gui.hours=ч
-replaymod.gui.minutes=мин
+replaymod.gui.minutes=мин.
```
Comment (Johni0702) [inline ru_RU.lang:69]: Is the dot really required/preferred when abbreviating "minutes"? Seems odd that it's only on minutes but not on hours, seconds and milliseconds.

File: ru_RU.lang [hunk 2]
```diff
@@ -209,35 +209,35 @@ replaymod.gui.rendersettings.addtoqueue=Добавить в очередь
...
 
 replaymod.gui.rendersettings.renderer.default=Стандартный
-replaymod.gui.rendersettings.renderer.stereoscopic=Стереоскопическая
+replaymod.gui.rendersettings.renderer.stereoscopic=Стереоскопич.
 replaymod.gui.rendersettings.renderer.cubic=Кубический
-replaymod.gui.rendersettings.renderer.equirectangular=Равнопромежуточная
+replaymod.gui.rendersettings.renderer.equirectangular=Равноугольный
 replaymod.gui.rendersettings.renderer.ods=ODS
-replaymod.gui.rendersettings.renderer.blend=Экспорт в Blender [Экспериментально]
+replaymod.gui.rendersettings.renderer.blend=Экспорт в Blender [БЕТА]
```
Comment (Johni0702) [inline ru_RU.lang:216]: Does "Beta" have a different connotation in Russian? Note that I would not want to call this feature Beta in English, it is very much only experimental, it doesn't even work with modern Blender versions, nor with modern Minecraft versions.

File: ru_RU.lang [hunk 3]
```diff
@@ -209,35 +209,35 @@ replaymod.gui.rendersettings.addtoqueue=Добавить в очередь
...
+replaymod.gui.rendersettings.customresolution.warning.cubic.height=Высота должна быть кратна 3
 replaymod.gui.rendersettings.bitrate=Битрейт
 replaymod.gui.rendersettings.framerate=Частота кадров
 replaymod.gui.rendersettings.chromakey=Хромакей
 replaymod.gui.rendersettings.nametags=Отображать имена
-replaymod.gui.rendersettings.includealpha=Сохранить альфа-канал
+replaymod.gui.rendersettings.includealpha=Сохранить Альфа-канал
 replaymod.gui.rendersettings.outputfile=Выходной файл
-replaymod.gui.rendersettings.sphericalFov=Сферическое поле зрения
+replaymod.gui.rendersettings.sphericalFov=Сферический FOV
```
Comment (Johni0702) [inline ru_RU.lang:235]: The game itself seems to be using the original value for this one:

Is there a good reason you changed it or should we stick to what Minecraft does?

File: ru_RU.lang (hunk 1 above)
Comment (xlifi) [reply ru_RU.lang:69]: Left by an accident

File: ru_RU.lang (hunk 2 above)
Comment (xlifi) [reply ru_RU.lang:216]: It's just shorter and obvious by context that it refers to replaymod function

File: ru_RU.lang (hunk 3 above)
Comment (xlifi) [reply ru_RU.lang:235]: I changed it because it probably doesn't fit in replaymod's ui

Comment (Johni0702) [reply ru_RU.lang:235]: It just about fits, so I'm gonna keep it:

File: ru_RU.lang [hunk 4]
```diff
@@ -209,35 +209,35 @@ replaymod.gui.rendersettings.addtoqueue=Добавить в очередь
...
 replaymod.gui.rendersettings.framerate=Частота кадров
 replaymod.gui.rendersettings.chromakey=Хромакей
 replaymod.gui.rendersettings.nametags=Отображать имена
-replaymod.gui.rendersettings.includealpha=Сохранить альфа-канал
+replaymod.gui.rendersettings.includealpha=Сохранить Альфа-канал
 replaymod.gui.rendersettings.outputfile=Выходной файл
 replaymod.gui.rendersettings.sphericalFov=Сферическое поле зрения
 replaymod.gui.rendersettings.sphericalmetadata=Сферические метаданные
-replaymod.gui.rendersettings.sphericalmetadata.error=Сферические метаданные возможно добавить только в видео формата MP4 с проекцией ODS или с равнопромежуточной проекцией
+replaymod.gui.rendersettings.sphericalmetadata.error=Сферические метаданные могут быть добавлены только к равноугольным видео в формате ODS или MP4
```
[the remark this answers is not in the document]
Comment (xlifi) [reply ru_RU.lang:237]: The best is to revert this change

**Comment by Johni0702** [comment]
Merged as 3fc91bc

**Pull request closed without merging**